### ISSUE TITLE
Return throttling params in server.json 

### ISSUE THREAD

**Issue by damian0815** [damian0815/drawify#56]
Return a couple of throttling params from http://drawify.onato.com/server.json to reduce bandwidth usage dynamically.

Currently server.json returns this:

```
{"server":"drawify2.onato.com"}
```

If it returned something like this:

```
{"server":"drawify2.onato.com",
 "throttle": "[number from 1-5 indicating requested update frequency throttling: 
    1 is full speed (check for updates every 30 seconds if it makes sense to do so), 
    5 is update only once a day]",
 "suggestedTimeout": "[cache timeout in seconds before checking for fresh data]"}
```

it would give us a way of making the app request updates less aggressively if the server is getting overwhelmed, avoiding whale situations. This may be overkill, I would imagine `throttle` would sit on `1` most of the time, but maybe nice to have 'just in case'.

**Comment by damian0815** [comment]
@onato what do you think? `priority/2` because it doesn't seem that important but feel free to bump it up.

**Comment by damian0815** [comment]
I am now doing very aggressive local caching, and making sure that I always go through the `nodes/updatedSince.json` interface before actually fetching the nodes.

Some statistics from a hopefully typical session this afternoon.

Browsing through the app normally, the calls to updatedSince.json are happening roughly once every 10 seconds (so 360 per hour) with a mean count of nodes passed of 24.7. Calls to nodes/view/x.json are happening about the same amount (0.1/second). User requests come through around 0.02/second -- for the moment, but this is mainly because there are only 4 users, it will increase depending on the node/user ratio. I'm not really doing any caching at all on the user.

This is with a cache timeout of 10s which is sort of ridiculous but make all the data passing happen very fast; like counts are propagated almost as fast as push notifications for example.

**Comment by damian0815** [comment]
**tl;dr** make sure that `nodes/updatedSince.json` is super duper optimized because i'm going to be hammering the shit out of it if I can.
